site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+
+{{< example class="p-lg-0 side-nav-example d-block d-lg-flex" >}}
+<button class="btn btn-primary d-lg-none" data-bs-target="#sideNavCollapsible" data-bs-toggle="offcanvas" aria-haspopup="true" aria-controls="sideNavCollapsible" id="sideNavCollapsibleToggle" onclick="boosted.Collapse.getOrCreateInstance('#collapsibleElement1').show()">Show basic collapsible side navigation</button>
+
+<div class="offcanvas-lg offcanvas-start" role="region" aria-labelledby="sideNavCollapsibleToggle" id="sideNavCollapsible">
+  <div class="offcanvas-header p-0">
+    <button type="button" class="btn-close ms-auto" data-bs-target="#sideNavCollapsible" data-bs-dismiss="offcanvas" aria-expanded="true" aria-controls="sideNavCollapsible"><span class="visually-hidden">Close</span></button>
+  </div>
+  <div class="offcanvas-body h-100 p-0">
+
+    <nav class="side-nav" aria-label="Collapsible side navigation inside an offcanvas">
+      <div class="side-nav-collapsible">
+        <div id="collapsibleElement1" class="collapse collapse-horizontal"><div>&nbsp;</div></div>
+        <button class="side-nav-toggle collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleElement1" aria-expanded="false" aria-controls="collapsibleElement1"><span class="visually-hidden">Open/Close side navigation</span></button>
+        <ul class="side-nav-content side-nav-scrollable list-unstyled m-0">
+          <li data-bs-toggle="tooltip" data-bs-title="Dashboard" data-bs-placement="right">
+            <a href="#" class="side-nav-item active fw-bold">
+              <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#dashboard"/>
+              </svg>
+              Dashboard
+            </a>
+          </li>
+          <li data-bs-toggle="tooltip" data-bs-title="Charts" data-bs-placement="right">
+            <a href="#" class="side-nav-item fw-bold">
+              <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#chart"/>
+              </svg>
+              Charts
+            </a>
+          </li>
+          <li data-bs-toggle="tooltip" data-bs-title="Financial" data-bs-placement="right">
+            <a href="#" class="side-nav-item fw-bold">
+              <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#financial"/>
+              </svg>
+              Financial
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+  </div>
+</div>
+
+<div class="d-none d-lg-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+{{< /example >}}
+
+### Collapsible with drawer content
```

Comment:
Should it be possible to close the drawer using `Esc` key? Maybe as an option?

Reply:
It'll come with the Js and the returns of a11y experts !